www/js/Controller/Post_item/Postitem_web.js

```diff
@@ -15,6 +15,21 @@ elephant.controller('PostitemController_web', function($scope,$localStorage ,$io
    */
   var imageToUpload = null;
 
+  var cropContainer = document.getElementById("crop-image-container");
```

Comment:
We have a class which wraps on the docuement.getElementBy__method__(). In order to maintain consistency, I would advice using that. It is located in www/js/Factory/Userinput.js. Example:

www/js/Controller/Form/Login.js
```
var login_data = {
      email: inputVal.getValue(EmailId),
      pass: inputVal.getValue(PassId)
    } 
```